core/com.b2international.snowowl.api.rest/src/com/b2international/snowowl/api/rest/admin/RepositoryRestService.java

```diff
@@ -69,7 +67,7 @@
 	public @ResponseBody DeferredResult<Repositories> getRepositories(
 			@ApiParam
 			@RequestParam(value="id", required=false)
-			List<String> idFilter) {
+			String[] idFilter) {
```

Comment:
It seems to work in `SnomedReferenceSetMemberRestService#search`, can't really tell why; information on this topic is contradictory.